### ISSUE TITLE
Linux Arch

### ISSUE THREAD

**Issue by mmtech2012** [ysc3839/FontMod#78]
Can I use this project in Arch Linux base OS?

**Comment by ysc3839** [comment]
No. It only works with Windows program. But it may work in [Wine](https://www.winehq.org/) environment.
